### PR TITLE
Nuxt 3 Dependency Updates

### PR DESCRIPTION
**Pull request by treckstar**
Uniform + Nuxt 3 is a great combo. Since I made these updates while testing I figured I would send them over.

Thanks!

### CODE REVIEW

**Comment by alexshyba** [comment]
Thank you so much for the first external contribution to our starters, @treckstar! Really appreciate it. Tagging my colleague @ilyasmez to help review this PR.

File: README.md [hunk 1]
```diff
@@ -9,7 +9,15 @@ https://www.npmjs.com/package/@uniformdev/uniform-nuxt
 
 ### Configure environment variables
 
-The project is already configured to use a sample Uniform project. Check out the [.env](./.env) file for instructions on how to use your own project instead.
+The project is already configured to use a sample Uniform project. Check out the [.env.example](./.env.example) file for instructions on how to use your own project instead.
+
+### Setup Uniform Preview URL for Visual Editing
+
+Make sure to [set the preview URL in the Uniform Canvas Settings](https://docs.uniform.app/docs/guides/composition/visual-editing/preview-mode#configure-the-preview-url-for-the-project) to the value for Nuxt.
```
Comment (ilyasmez) [inline README.md:16]: ❤️

File: nuxt.config.ts [hunk 2]
```diff
@@ -8,9 +8,9 @@ export default defineNuxtConfig({
   uniform: {
     projectId: process.env.UNIFORM_PROJECT_ID,
     readOnlyApiKey: process.env.UNIFORM_API_KEY,
-    apiHost: process.env.UNIFORM_CLI_BASE_URL,
-    edgeApiHost: process.env.UNIFORM_CLI_BASE_EDGE_URL,
-    outputType: process.env.OUTPUT_TYPE,
+    apiHost: process.env.UNIFORM_CLI_BASE_URL || "https://uniform.app",
+    edgeApiHost: process.env.UNIFORM_CLI_BASE_EDGE_URL || "https://uniform.global",
+    outputType: process.env.OUTPUT_TYPE || 'standard',
```
Comment (ilyasmez) [inline nuxt.config.ts]: These are already set as default values on the module-level. Did you face any issues with that?

Or you want to handle empty strings? If that's the case, you can just set the falsy values to undefined (using `|| undefined,`) to leverage the module's defaults.

File: .nvmrc [hunk 3]
```diff
@@ -1 +1 @@
-v16.15.0
\ No newline at end of file
+v18.18.0
```
Comment (ilyasmez) [inline .nvmrc:1]: ❤️

**Review by ilyasmez — COMMENTED** [review]
Thank you very much for taking the time to open a PR, @treckstar!! ⭐

The PR looks great! We just need to fix 3 things before merging:
- Bring back the `.env` file: We need to bring back the .env file because it's used for an automated deployment of this project (yes, we could create one out of the .env.example, but that's not implemented yet.)
- gitignore the lock file: This goes against the best practices for sure, but it's required for the internal deployment of the repo (in addition to some issues related to running the project in StackBlitz).
- Use `npm` instead of `pnpm`: We're big fans of pnpm and we use it a lot internally, but to keep the project more accessible, we prefer to use the standard package manager.

File: nuxt.config.ts (hunk 2 above)
Comment (treckstar) [reply nuxt.config.ts]: Done!

**Comment by treckstar** [comment]
Thank you for taking the time to review! Much appreciated :)

I think I touched on all of those changes. I did pull down and it builds locally. I think Netlify needs to clear cache? I also got that error, and once `rimraf node_modules` was done, it worked just fine. 

If you'd like me to clean this branch up and rebase to 1 commit or anything at all. Just let me know!

**Review by ilyasmez — APPROVED** [review]
Perfect! Thank you very much for your contribution @treckstar! 🙏 

And yes, clearing the cache on Netlify did the job, your PR was successfully deployed here: https://deploy-preview-62--uniformconf-nuxt.netlify.app/

**Pull request merged**